$MirOS: ports/emulators/qemu/patches/patch-block-qcow_c,v 1.2 2005/11/15 12:25:08 tg Exp $
--- block-qcow.c.orig	Sun Sep  4 17:11:09 2005
+++ block-qcow.c	Tue Nov 15 11:52:48 2005
@@ -683,6 +683,5 @@ BlockDriver bdrv_qcow = {
     qcow_create,
     qcow_is_allocated,
     qcow_set_key,
+    NULL
 };
-
-
